Corona-Warn-App/src/main/java/de/rki/coronawarnapp/task/TaskController.kt

```diff
@@ -63,15 +63,15 @@ class TaskController @Inject constructor(
     }
 
     /**
-     * Don't re-use taskrequests, create new requests for each submission.
+     * Don't re-use task requests, create new requests for each submission.
      * They contain unique IDs.
      */
     fun submit(request: TaskRequest) {
         if (!taskFactories.containsKey(request.type.java)) {
             throw MissingTaskFactoryException(request::class)
         }
         Timber.tag(TAG).i("Task submitted: %s", request)
-        taskQueue.offer(request)
+        taskQueue.trySend(request).isSuccess
```

Comment:
🤔  calling `isSuccess` is not necessary here?

Reply:
I guess the auto-migration suggested it, since `offer()` returns a boolean. However, we don't use that boolean so we don't have to call `.isSuccess`

Reply:
`trySend().isSuccess` returns `Boolean`
`trySend()` returns `ChannelResult`

But since the return type is not relevant in this situation, I don't see a reason to call `.isSuccess`

Reply:
Ok, only for you, I will change it 🎁

Reply:
❤️